Plugins/Statistics/GitStatistics/LineCounter.cs

```diff
@@ -20,7 +20,7 @@ public sealed class LineCounter
 
         public void FindAndAnalyzeCodeFiles(string filePattern, string directoriesToIgnore, IEnumerable<string> filesToCheck)
         {
-            var extensions = filePattern.Replace("*", "").Split(';').ToHashSet(StringComparer.InvariantCultureIgnoreCase);
+            var extensions = LinqExtensions.ToHashSet(filePattern.Replace("*", "").Split(';'), StringComparer.InvariantCultureIgnoreCase);
```

Comment:
Why this change?

Reply:
A name conflict, .NET Runtime added a new method `ToHashSet` with the same signature as ours. However it wasn't a straight forward replace/deletion in our codebase, so I just left it as is for now.